src/content/en/fundamentals/performance/optimizing-content-efficiency/client-hints/index.md

```diff
@@ -0,0 +1,680 @@
+project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: TBD
+
+{# wf_updated_on: 2018-11-11 #}
+{# wf_published_on: 2018-11-12 #}
```

Comment:
Let's update these dates to a week from now (to give us enough time to sort out the CDS text issues and final reviews)

Reply:
I updated the published on field, but the updated on field auto updates on commit/push/whatnot. Otherwise, this should be GTG.